libfreerdp/core/gateway/arm.c

```diff
@@ -311,6 +318,319 @@ static char* arm_create_request_json(rdpArm* arm)
 	return message;
 }
 
+static WINPR_CIPHER_CTX* treatAuthBlob(const BYTE* pbInput, size_t cbInput)
+{
+	WINPR_CIPHER_CTX* ret = NULL;
+	char algoName[100];
```

Comment:
maybe initialize with `={0};`

Reply:
No need to zero the full array, we do set the end of string later.